spec/features/openid_connect/redirect_uri_validation_spec.rb

```diff
@@ -74,11 +74,6 @@
 
       click_on t('links.back_to_sp', sp: 'Test SP')
       expect(current_url).to eq(sp_redirect_uri)
-
-      visit new_user_session_path(request_id: '123', redirect_uri: 'evil.com')
-
-      click_on t('links.back_to_sp', sp: 'Test SP')
-      expect(current_url).to eq(sp_redirect_uri)
```

Comment:
It looks like this test was added as a regression spec for a security feature, are we sure it should be removed, or can it be updated to handle the new user flow?

Reply:
Also, I think it could be a good idea to have some feature spec coverage for cancelling at this screen (i.e. that the user arrives back at the SP if they choose to cancel at the completions step).

Reply:
This was removed in error.

I agree on the feature spec coverage.